client/src/util.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { window, workspace, ColorThemeKind } from "vscode";
+
+export function wrapCode(code: string): string {
+  const outputHtml = !!workspace
+    .getConfiguration("SAS")
+    .get("session.outputHtml");
+  const htmlStyle: string = workspace
+    .getConfiguration("SAS")
+    .get("results.html.style");
+
+  if (outputHtml) {
+    let odsStyle = "";
+    switch (htmlStyle) {
+      case "(auto)":
+        switch (window.activeColorTheme.kind) {
+          case ColorThemeKind.Light: {
+            odsStyle = "Illuminate";
+            break;
+          }
+          case ColorThemeKind.Dark: {
+            odsStyle = "Ignite";
+            break;
+          }
+          case ColorThemeKind.HighContrast: {
+            odsStyle = "HighContrast";
+            break;
+          }
+          case ColorThemeKind.HighContrastLight: {
```

Comment:
nit: how about `default:`

Reply:
If window.activeColorTheme.kind does not match one of the cases/enums for ColorThemeKind, then odsStyle will remain "" (what is initialized to), thus no style= statement will be generated and the server default style will be used. This seems reasonable to me. I'll go ahead and add default though, so it's more explicit and readable.